.github/workflows/runSchematronValidation.yml

```diff
@@ -55,16 +65,25 @@ jobs:
           schematronRoot=${schematronName%.*}
           echo "schematronRoot: ${schematronRoot}"
           echo "schematron code ${{ github.workspace }}/resources/validations/lib/schematron/trunk/schematron/code/iso_svrl_for_xslt2.xsl"
-          mkdir /tmp/target
-          rm -rf /tmp/target/*.xsl;
-          `java -cp ${SAXON_CP} net.sf.saxon.Transform -o:/tmp/target/${schematronRoot}.xsl -s:${{ github.workspace }}${qualifiedSchematronName} ${{ github.workspace }}/resources/validations/lib/schematron/trunk/schematron/code/iso_svrl_for_xslt2.xsl`
-          
+          rm -rf ${{ github.workspace }}/resources/validations/target/*.xsl
+
+          java -cp ${SAXON_CP} net.sf.saxon.Transform \
+          -o:"${{ github.workspace }}/resources/validations/target/${schematronRoot}".xsl \
+          -s:"${{ github.workspace }}/${qualifiedSchematronName}" \
+          ${{ github.workspace }}/resources/validations/lib/schematron/trunk/schematron/code/iso_svrl_for_xslt2.xsl \
+          allow-foreign=true
```

Comment:
Are we going to parameterize this like the script?

Reply:
yes, we will eventually just call the .sh as you suggested when we paired